google/ggrc-core
# GGRC-5459 Tabs in HNB are displayed with latency on the Audit page

# Issue description

*This Pull Request is a continuation of https://github.com/google/ggrc-core/pull/8452*

Tabs in HNB are displayed with latency on the Audit page

# Steps to test the changes

1. Open any Audit page with Assessment and look at the HNB.
2. Open Assessment in a separate tab.

**Expected Result:** Improve performance of displaying Tabs in HNB on the Audit page

# Solution description

**BE:** Add separate endpoint that returns only snapshot count by snapshot type. Implemented for Audit, Assessment and Issue. Url: `api/<audits or assessments or issues>/<id>/snapshot_counts`
**Usage**: snapshot_counts endpoint expects json in request data: `{"snapshot_types": [<Snapshot type 1>,
<Snapshot type 2>, ..., <Snapshot type N>]}` and returns:
`{<Snapshot type 1>: <count 1>, ..., <Snapshot type N>: <count N>}`

# Sanity checklist




# PR Review checklist
